content/docs/integrations/kubernetes/index.md

```diff
@@ -456,6 +323,97 @@ CA_CERT:/home/node/my-cluster-cert.crt
 
 If you are running into issues with using self-signed certificates, you can use Node.js environment options to tell the underlying Node.js process how to handle the certificate. The relevant options are [explicitly defining extra certificate files](https://nodejs.org/api/cli.html#cli_node_extra_ca_certs_file) or alternatively [turning off self-signed certificate validation](https://docs.snyk.io/features/snyk-broker/set-up-snyk-broker/how-to-install-and-configure-your-snyk-broker-client#disable-certificate-verification). 
 
+#### Running in a k8s pod
+
+If you would like to run the broker container within a pod in your cluster, you will need to create a service account for the pod that would allow it to access the resources neccessary to operate the backstage plugin. Here is an example of such a manifest. Please note to replace the "$TENANT_NAME" with your Roadie tenant name.
+
+<details>
+
+<Summary>Show broker-kube-manifest.yaml</Summary>
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: read-stuff
+rules:
+  - apiGroups: [""]
+    resources:
+      - pods
+      - configmaps
+      - services
+      - deployments
+      - replicasets
+      - horizontalpodautoscalers
+      - ingresses
+    verbs: 
+      - "get"
+      - "list"
+      - "watch"
+  - apiGroups: ["batch"]
+    resources:
+      - jobs
+      - cronjobs
+    verbs: 
+      - "get"
+      - "list"
+      - "watch"
+  - apiGroups: ["extensions", "apps", "autoscaling", "networking.k8s.io"]
+    resources: ["deployments", "ingresses", "replicasets", "horizontalpodautoscalers"]
```

Comment:
We are mixing two different kinds of list formats here and the `resources` block above. Let's use one only